test/game_statistics_test.rb

```diff
@@ -0,0 +1,61 @@
+require "./lib/game_statistics"
+require "minitest/autorun"
+require "minitest/pride"
+
+class GameStatisticsTest < MiniTest::Test
+
+  def setup
```

Comment:
@ajtran303 setup and test it exists with attributes are the same across all inheriting classes. But we still have to test it for each specific class to see if it is actually inheriting it from StatTracker